docs/documentation/upgrading/topics/changes/changes.adoc

```diff
@@ -1,6 +1,10 @@
 [[migration-changes]]
 == Migration Changes
 
+=== Migrating to 26.5.0
+
+include::changes-26_5_0.adoc[leveloffset=2]
```

Comment:
We should not include it here for now as once 26.4.0 is out, there's no 26.5.0 release.

Reply:
I still see the 26.5.0 milestone. I'll put on hold further doc changes until the release target is clear.

Reply:
Ahh, you're right.